Students/Benny/3. Protocols and bridging/Questions.md

```diff
@@ -20,3 +28,7 @@
 			[self.collectionView reloadData];
 		}];
 ```
+
+3. Explain the difference between NSOperation and GDC please.
+
+4. NSDictionary 'bindings' argument in NSPredicate filtering?
```

Comment:
If you still need answer to this question please ask it again. I don't quite get the question now.